diff --git a/lib/makeWebpackConfig.js b/lib/makeWebpackConfig.js
index 62615b1c8db5a13a19ff960017aecfc6f0950219..a9602a4819ac43bf75f4510fd475e3d30642b17e 100644
--- a/lib/makeWebpackConfig.js
+++ b/lib/makeWebpackConfig.js
@@ -138,7 +138,7 @@ module.exports = async (playroomConfig, options) => {
         },
         {
           test: /\.css$/,
-          include: path.dirname(require.resolve('codemirror/package.json')),
+          include: [path.dirname(require.resolve('codemirror/package.json'))],
           use: [MiniCssExtractPlugin.loader, require.resolve('css-loader')],
         },
         {
@@ -167,7 +167,7 @@ module.exports = async (playroomConfig, options) => {
         chunksSortMode: 'none',
         chunks: ['index'],
         filename: 'index.html',
-        favicon: path.join(__dirname, '../images/favicon.png'),
+        // favicon: path.join(__dirname, '../images/favicon.png'),
         base: playroomConfig.baseUrl,
       }),
       new HtmlWebpackPlugin({
diff --git a/src/Playroom/Playroom.css.ts b/src/Playroom/Playroom.css.ts
index 84c166f0a4dfd14e52461c7d105411923603399a..357c8fb08c0617f8a129acb35dbb4c24a351609f 100644
--- a/src/Playroom/Playroom.css.ts
+++ b/src/Playroom/Playroom.css.ts
@@ -15,6 +15,7 @@ globalStyle('html[data-playroom-dark]', {
 
 globalStyle('body', {
   margin: 0,
+  backgroundColor: 'inherit !important',
 });
 
 export const root = sprinkles({
